src/main/java/lotto/domain/LottoNumberGenerator.java

```diff
@@ -0,0 +1,24 @@
+package lotto.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoNumberGenerator {
+
+    private static final int LOTTO_MIN_NUMBER = 1;
+    private static final int LOTTO_MAX_NUMBER = 45;
+    private static final int LOTTO_MAX_SIZE = 6;
+    private static List<LottoNumber> lottoList;
+
+    static {
+       lottoList = IntStream.range(LOTTO_MIN_NUMBER, LOTTO_MAX_NUMBER)
+           .boxed().map(LottoNumber::new).collect(Collectors.toList());
```

Comment:
아래와 같이 변경할 수 있을 것 같아요! 😃
```suggestion
    private static final List<LottoNumber> LOTTO_NUMBERS;

    static {
       LOTTO_NUMBERS = IntStream.range(LOTTO_MIN_NUMBER, LOTTO_MAX_NUMBER)
                .mapToObj(LottoNumber::new)
                .collect(Collectors.toList());
```

stream 함수의 경우 꼭 한 줄에 하나의 메서드를 호출할 수 있도록 해주세요! (가독성 + 인텔리제이의 편의 기능을 이용할 수 있어요! 😉 )
<img width="676" alt="image" src="https://user-images.githubusercontent.com/47850258/153246955-d2b81fe5-05b8-4069-bb41-ffd22ed21cf3.png">

Reply:
반영하였습니다 !